scripts/sct_utils.py

```diff
@@ -611,9 +611,12 @@ def check_if_3d(fname):
     :return: True or False
```

Comment:
The docstring says "3d or less" --> that's why I used `<= 3` instead of `==`. But we may want to be more specific by implementing a `check_if_2d()` function instead !?

Reply:
i would not create another function, but generalize the current function. E.g., calling it: 
```python
check_dim(fname, dim=[3])
Check if input dimension matches the input dimension requirements specified in the dim list. 
Example: to check if an image is 2D or 3D: check_dim(my_file, dim=[2, 3])
```